go/client/chat_cli_utils.go

```diff
@@ -125,6 +126,9 @@ func parseConversationResolver(ctx *cli.Context, tlfName string) (resolver conve
 	if resolver.TopicType, err = parseConversationTopicType(ctx); err != nil {
 		return resolver, err
 	}
+	if resolver.TopicType == chat1.TopicType_CHAT && len(resolver.TopicName) != 0 {
+		return resolver, errors.New("multiple topic names are not yet supportted with chat conversations")
```

Comment:
error message is misspelled. Just say `"multiple topic names are not yet supported"`

How does one get into this situation? Will the conversation be broken in this case?  Have you seen i t?